internal/client.go

```diff
@@ -492,23 +492,37 @@ type (
 		Authority string
 
 		// Enables keep alive ping from client to the server, which can help detect abruptly closed connections faster.
+		//
+		// Deprecated: Keep alive is now enabled by default, to disable set DisableKeepAliveCheck.
```

Comment:
I'd be tempted to literally break compatibility and just remove this field. There's a difference between "deprecated" as if it is still used and "unused field". Thoughts?

Reply:
Yeah I thought about removing it, my only issue with removing it is I think it will only break the people who are not effected by this change. Since if you wanted it false you probably just didn't set it. I am fine removing it though if you want to

Reply:
Let's bring up with team if it can wait a day or so

Reply:
Fine with me